### ISSUE TITLE
Add a parallel subnetwork along the existing network

### ISSUE THREAD

**Issue by thechidiya** [fizyr/keras-retinanet#1330]
Hello,
This is not really an issue, just trying to customize retinanet. 

I am using retinanet to identify humans and then determine their ages. What I did here is that I've added a subnet along with regression and classification subnet which will predict the age of the person in a given image. If I train retinanet without this extra subnet it gives me good accuracy. But when I train using this extra subnet I'm hurting mAP. does anyone has encountered this task of adding a parallel subnet along with regression and classification subnet? I suspect it has something to do with the fact that I'm training many outputs at the same time? I tried adding weights to the losses but still no luck!!

Any help would be appreciated! :) 

Thanks!
